project/gh-action/infrastructure/kubernetes/deployment.yaml

```
@@ -0,0 +1,34 @@
+  replicas: "3"  
+  selector:
+    matchLabels:
+      app: frontend 
+  template:
+    metadata:
+      labels:
+        app: web 
+    spec:
+      containers:
+      - name: web-app
+        image: nginx  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)